Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Edit_polyhedron_plugin.cpp

```diff
@@ -83,9 +99,25 @@ QList<QAction*> Polyhedron_demo_edit_polyhedron_plugin::actions() const {
 bool Polyhedron_demo_edit_polyhedron_plugin::applicable(QAction*) const { 
   Q_FOREACH(CGAL::Three::Scene_interface::Item_id i, scene->selectionIndices())
   {
-    if(qobject_cast<Scene_polyhedron_item*>(scene->item(i)) 
-        || qobject_cast<Scene_edit_polyhedron_item*>(scene->item(i)))
+    if(qobject_cast<Scene_facegraph_item*>(scene->item(i)))
       return true;
+    Scene_edit_polyhedron_item* edit_item = qobject_cast<Scene_edit_polyhedron_item*>(scene->item(i));
+    if(!edit_item)
+      return false;
+    if(edit_item->poly_item())
+    {
+      if (qobject_cast<Scene_facegraph_item*>(edit_item->poly_item()))
+      {
+        return true;
+      }
+    }
+    else
+    {
+      if (qobject_cast<Scene_facegraph_item*>(edit_item->sm_item()))
+      {
+        return true;
+      }
+    }
```

Comment:
Yes, it is because the Scene_selection_item is split into two targets, and so we also must have two targets for the edit_plugin, which was not conceived to do so. We therefore have two identical targets with two different names.